tests/ui/enum_variants.stderr

```diff
@@ -1,97 +1,97 @@
 error: Variant name ends with the enum's name
-  --> $DIR/enum_variants.rs:24:5
+  --> $DIR/enum_variants.rs:25:5
    |
-24 |     cFoo,
+25 |     cFoo,
    |     ^^^^
    |
    = note: `-D clippy::enum-variant-names` implied by `-D warnings`
 
 error: Variant name starts with the enum's name
-  --> $DIR/enum_variants.rs:35:5
+  --> $DIR/enum_variants.rs:36:5
    |
-35 |     FoodGood,
+36 |     FoodGood,
    |     ^^^^^^^^
 
 error: Variant name starts with the enum's name
-  --> $DIR/enum_variants.rs:36:5
+  --> $DIR/enum_variants.rs:37:5
    |
-36 |     FoodMiddle,
+37 |     FoodMiddle,
    |     ^^^^^^^^^^
 
 error: Variant name starts with the enum's name
-  --> $DIR/enum_variants.rs:37:5
+  --> $DIR/enum_variants.rs:38:5
    |
-37 |     FoodBad,
+38 |     FoodBad,
    |     ^^^^^^^
 
 error: All variants have the same prefix: `Food`
-  --> $DIR/enum_variants.rs:34:1
+  --> $DIR/enum_variants.rs:35:1
    |
-34 | / enum Food {
-35 | |     FoodGood,
-36 | |     FoodMiddle,
-37 | |     FoodBad,
-38 | | }
+35 | / enum Food {
+36 | |     FoodGood,
+37 | |     FoodMiddle,
+38 | |     FoodBad,
+39 | | }
```

Comment:
By the way, why doesn't clippy use `-Z ui-testing`?
In the compiler it's used quite successfully to avoid this kind of churn in UI test outputs.

Reply:
I never knew this flag existed. But yes, Clippy should use this flag.